src/scores/pandas/continuous.py

```diff
@@ -0,0 +1,98 @@
+"""
+Import the functions from the implementations into the public API
+"""
+
+from scores import continuous as __continuous
+from scores.pandas.typing import PandasType
+
+
+def mse(
+    fcst: PandasType,
+    obs: PandasType,
+    angular: bool = False,
+) -> PandasType:
+    """Calculates the mean squared error from forecast and observed data.
+
+    A detailed explanation is on [Wikipedia](https://en.wikipedia.org/wiki/Mean_squared_error)
+
+
+    Dimensional reduction is not supported for pandas and the user should
+    convert their data to xarray to formulate the call to the base metric, `scores.continuous.mse`.
+
+    Args:
+        fcst: Forecast or predicted variables in pandas.
```

Comment:
Do we also want the option of supplying a pd.DataFrame and specifying the names of the fcst/obs columns?

Reply:
That would be a useful feature,
Could be done through a decorator?

Reply:
See #161 for implementation

Reply:
I prefer the previous version for a number of reasons. For reference, plotly express implements an interface which accepts either series, or a dataframe plus strings for the column names. My reasoning is as follows:

1. It's simpler to just have one way to do things. It's really not hard for a user to just pass in the columns from a dataframe.
2. In this implementation, it's assumed the column names will be 'fcst' and 'obs'. It is probably not going to be uncommon to have column names which match the actual name of the forecast source rather than just being called 'fcst'. I think the plotly API is easier where the dataframe mode is e.g. " function(df, x="name", y="name2"). 

In terms of upsides, the advantage is that  I think if we don't have a dataframe based API, people will ask for one. 

But I think it's more explicit to pass in the series. As seen in the tests below, this is as simple as passing in df['fcst'] and df['obs']. This is also in my mind more 'literate' / self-documenting. It is very hard in this way to get unexpected behaviour, it's completely clear as the function is being called what is being passed in.

Reply:
I'm planning to merge the version without the dataframe API, and then suggest moving a dataframe option into a future merge request.